packages/docs/pages/index/rating.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <docs-content :config="config"></docs-content>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+import DocsContent from '../../components/docs-content/DocsContent'
+import ratingConfig from '../../components/docs-content/configs/rating'
+
+@Component({
+  components: {
+    DocsContent,
+  },
+})
+export default class Rating extends Vue {
+  get config () {
+    return ratingConfig
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+h1,
```

Comment:
I think we mostly want to use default framework styling. Are these overrides really needed?

Reply:
Why did we make this reset in ui then?